.circleci/config.yml

```diff
@@ -53,6 +52,11 @@ references:
     environment:
       <<: *infrastructure_config_environment
 
+  upload_container_config: &upload_container_config
```

Comment:
What's the point of this additional config ?

Reply:
it is because the Upload instances could use different (smaller) resource than build